tests/tests.cpp

```diff
@@ -20,6 +22,10 @@ TEST(tests, issueBar){
 	bool returnValue = func_1;
 	EXPECT_EQ(true, returnValue);
 }
+TEST(tests, issue75){
```

Comment:
I see here you did not put comments above your tests. Please follow Eric Bucks Words example & put Doxygen informational comments above your tests for proper documentation.